appengine/datastore/README.md

```diff
@@ -8,12 +8,35 @@ is saved using the Dataset `save` method. Then, the ten most recent
 visits are retrieved in descending order by building a Query, and
 using the Dataset `run` method.
 
-## Run
+## Run Locally
+
+1. Make sure `gcloud` is installed. 
```

Comment:
Update 1. to:

1. Install and initialize [Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts).